src/components/calculator.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+class CalcUserInterface extends React.Component { // eslint-disable-line
+  render() {
+    return (
+      <section>
+        <div className="container">
```

Comment:
- [ ] Nice work implementing the `Calculator` component :clap: However it would be nice to use **functional components** instead of **class components** as requested in the project [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md)

Reply:
Okay, thanks and noted.